spec/fixtures/lexis_nexis_responses/true_id_response_failure_no_liveness.json

```diff
@@ -626,7 +626,7 @@
         },
         {
           "Group": "IDAUTH_FIELD_DATA",
-          "Name": "Fields_xpirationDate_Day",
+          "Name": "Fields_ExpirationDate_Day",
```

Comment:
cc @amathews-fs @jmhooper do we know if this field was copy-pasted wrong somewhere on our end? Or if it's really in the API like this? I really hope it's on our side 😬

Reply:
Nope! As I recall that's direct from their API and I laughed at it.

Reply:
![oh no](https://media.giphy.com/media/J3MJAf2FbKO8oaTaTv/giphy.gif)

Reply:
98faf08d2447f94cfbc339b30e659aefd0c32cb6 😭

Reply:
gives me `idpxtra` vibes

Reply:
We may be able to clean that up in the future by using a different field they are passing back since there are semi-duplicate structures.